pkg/operator/starter.go

```diff
@@ -173,7 +174,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	// we keep the default behavior of exiting the controller once a gate changes
 	featureGateAccessor.SetChangeHandler(featuregates.ForceExit)
 
-	operatorClient, dynamicInformers, err := genericoperatorclient.NewStaticPodOperatorClient(controllerContext.KubeConfig, operatorv1.GroupVersion.WithResource("etcds"), operatorv1.GroupVersion.WithKind("Etcd"), ExtractStaticPodOperatorSpec, ExtractStaticPodOperatorStatus)
+	operatorClient, dynamicInformers, err := genericoperatorclient.NewStaticPodOperatorClient(clock.RealClock{}, controllerContext.KubeConfig, operatorv1.GroupVersion.WithResource("etcds"), operatorv1.GroupVersion.WithKind("Etcd"), ExtractStaticPodOperatorSpec, ExtractStaticPodOperatorStatus)
```

Comment:
do you mind to split out the library-go update into a separate PR? we can merge that earlier, I see there is a bunch of static pod changes there too, which we might want to separate out - just in case they mess up CI somehow.

Reply:
oh you already got #1365 - resolved :) thanks!